extensions/amp-story/1.0/live-story-manager.js

```diff
@@ -46,16 +46,11 @@ export class LiveStoryManager {
    * configuration and appends it to the DOM.
    */
   build() {
-    const listId = userAssert(
-      this.storyEl_.getAttribute('dynamic-live-list'),
-      'amp-story element must contain the dynamic-live-list attribute to ' +
-        'use the live story functionality.'
-    );
     const liveListEl = createElementWithAttributes(
       this.ampStory_.win.document,
       'amp-live-list',
       dict({
-        'id': listId,
+        'id': 'i-amphtml-' + this.storyEl_.id + '-dynamic-list',
```

Comment:
Just to clarify: I imagine this is going to work even if the story has no ID because it'd evaluate to `undefined`, but wouldn't the line 248 in the `live-list-manager.js` fail?

Reply:
Discussed offline: we still need the amp-story element to have an ID, since that is what we use to find our custom "items slot" at the live-list level.

We also check for its presence at the live-story-manager.js level.